lib/parse.js

```diff
@@ -142,7 +142,7 @@ class Parser {
         }
 
         if (items.length > 1) {
-            if (items.every((item) => item.type === 'Relation')) {
+            if (items.every((item) => item.type === 'Apply')) {
```

Comment:
This should probably be a bit more restrictive.  It should also check that the `op` for each of these is one of `eq, lt, gt, le, ge`.